api/internal/localizer/localizer.go

```diff
@@ -141,8 +132,47 @@ func (lc *localizer) localizeNativeFields(kust *types.Kustomization) error {
 		}
 	}
 
-	// TODO(annasong): localize all other kustomization fields: resources, bases, crds, configurations,
-	// openapi, patchesJson6902, patchesStrategicMerge, replacements
+	for name, patches := range map[string][]types.Patch{
+		"patches": kust.Patches,
+		// Allow use of deprecated field
+		//nolint:staticcheck
+		"patchesJson6902": kust.PatchesJson6902,
+	} {
+		for i := range patches {
+			if patches[i].Path != "" {
+				newPath, err := lc.localizeFile(patches[i].Path)
+				if err != nil {
+					return errors.WrapPrefixf(err, "unable to localize %s path %q", name, patches[i].Path)
+				}
+				patches[i].Path = newPath
+			}
+		}
+	}
+	//nolint:staticcheck
+	for i, patch := range kust.PatchesStrategicMerge {
+		_, isFile, err := lc.loadResource(string(patch))
+		if err != nil {
+			return errors.WrapPrefixf(err, "invalid patchesStrategicMerge entry")
```

Comment:
I didn't include the path in these error messages because the path was passed to the method that returned `err`, and so `err` should include the path in its error message. Let me know if I should include the path again in the prefix.

Reply:
sg. I was trying to ask why not including path here.